jbpm-flow/src/main/java/org/jbpm/process/core/impl/ProcessImpl.java

```diff
@@ -49,7 +51,7 @@
     private ContextContainer contextContainer = new ContextContainerImpl();
     private Map<String, Object> metaData = new HashMap<String, Object>();
     private transient Map<String, Object> runtimeMetaData = new HashMap<String, Object>();
-    private Set<String> imports;
+    private Set<String> imports = new HashSet<>(Arrays.asList("org.kie.api.runtime.process.CaseData", "org.kie.api.runtime.process.CaseAssignment"));
```

Comment:
What if I add other custom imports to my process definition? It seems that it will override these automatic imports too, won't it?

Reply:
Hmm, the only way I can change imports field is by setImports() method which will override this initialization. But maybe it cannot happen, just wanted to discuss it.

Reply:
Thanks, now I see that there is `list.add(name);` on a list returned by getter.